pandas/tests/frame/test_block_internals.py

```diff
@@ -45,7 +45,7 @@ def test_setitem_invalidates_datetime_index_freq(self):
         ts = dti[1]
 
         df = DataFrame({"B": dti})
-        assert df["B"]._values.freq == "D"
+        assert df["B"]._values.freq is None
```

Comment:
this change kind of makes the test useless? 

(and is this a breaking change? I am not fully up to date with out freq support for columns (non-index))

Reply:
> (and is this a breaking change? I am not fully up to date with out freq support for columns (non-index))

Not exactly.  We dont care much about the freq while this is in a column, we mainly want to make sure if we extract the values we correctly have freq=None

Trouble is that doesn't do anything for pre-existing views.  #31218 ultimately the correct solution is to remove freq from the DTA/TDA and only have it on DTI/TDI.

Reply:
@jorisvandenbossche i think this is the main outstanding question (fastparquet docbuild notwithstanding).  thoughts on how to handle this?

Reply:
@jorisvandenbossche gentle ping on this comment